### ISSUE TITLE
Is there a way to change the filename for __temp__.pov to enable it to run in parallel sessions?

### ISSUE THREAD

**Issue by metalaureate** [Zulko/vapory#59]
I am trying to run multiple provray sessions in parallel (off a job queue), but they conflict over` __temp__.pov`--when rendering, this file is locked and cannot be opened by a parallel instance. I was wondering if anyone here knew how to make povray operate with non-conflicting temp file resources.

**Comment by snellejelle99** [comment]
Hi if your are running povray 3.7 then rendering is already done with multi threading and will utilize your cpu 100%. Opening more instances of povray won't make it go significantly  faster since 2 instances will then each get 50%. I therefore suggest that you complete your job queue sequentially.

**Comment by kumari-jaya** [comment]
You can try running it sequentially like in a loop . As mentioned by @snellejelle99 , it might crash your system .

**Comment by rojizo** [comment]
You can add tempfile named argument to the render method of the scene

```scene.render("image.png", width=1920, height=1080,antialiasing=0.001,tempfile="filename")```

I'm using this to use of a cluster to build a video
